### DISCUSSION TITLE
A deficiency of which vitamin can cause scurvy?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#443] [Galaxy Trivia] [answered]
**Bonus:** How many human essential vitamins in total?

**Comment by peterbaccaro** [comment, marked as answer]
Scurvy is caused by not having enough **vitamin C** in your diet for at least 3 months.
There are **13 essential vitamins** - vitamins A, C, D, E, K, and the B vitamins (thiamine, riboflavin, niacin, pantothenic acid, biotin, B6, B12, and folate).
